templates/etc/httpd/conf.d/pulp_rpm.conf.erb

```diff
@@ -2,17 +2,32 @@
 # Apache configuration file for pulp web services and repositories
 #
 
-# -- HTTPS Repositories ---------
-Alias /pulp/repos /var/www/pub/yum/https/repos
+AddType application/x-pkcs7-crl .crl
+AddType application/x-x509-ca-cert .crt
```

Comment:
Is this needed here? I saw it also in https://github.com/Katello/puppet-katello/pull/98/files#diff-40567eb5f7aff653485984fef811556cR9 or maybe the question is the other way around?